File: 2485-find-the-pivot-integer/2485-find-the-pivot-integer.py
class Solution:
    def pivotInteger(self, n: int) -> int:
        sum_n= (n**2+n)//2
        left=0
        for i in range(1, n+1):
            right= sum_n-left-i
            if (left==right):
                return i
            left= left+i
        return -1

File: 2485-find-the-pivot-integer/2485-find-the-pivot-integer.cpp
class Solution {
public:
    int pivotInteger(int n) {
        int total= ((n)*(n+1))/2;
        int left=0;
        for (int i=1; i<n+1; i++){
            int right= total-left-i;
            if (left==right){
                return i;
            }
            left= left+i;
        }
        return -1;
    }
};